### PR TITLE
Created Badge component

### PR DESCRIPTION
**Pull request by anushasunil**
Created badges for the avatar, different types of buttons and also text.

https://user-images.githubusercontent.com/90134948/152190198-050236e8-a278-4905-b60d-b713d4b9eb82.mp4

circle avatar with badges
squircle with badges
buttons with badges
text with badges

### CODE REVIEW

File: Stylesheets/Components/badge.css [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+.squircle + .badge {
+    right: -0.5rem;
+    bottom: -0.5rem;
+}
+
+.squircle.xsmall + .badge {
+    right: -0.2rem;
+    bottom: -0.2rem;
+}
+
```
Comment (raman1999) [inline Stylesheets/Components/badge.css:30]: Everything else look good , just try to use px for units less than 1rem

Comment (anushasunil) [reply Stylesheets/Components/badge.css:30]: okay, I'll do the changes. thankyou.

File: Stylesheets/Components/badge.css [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    font-weight: 700;
+}
+
+.badge.notification.text {
+    top: -12px;
+    right:-12px;
+}
+
+.notification.text.large {
+    top: -1.2rem;
```
Comment (vivekmehto) [inline Stylesheets/Components/badge.css]: adjust top so that 'e' of text didn't get hidden.

File: Stylesheets/Components/badge.css [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+}
+
+.badge.dnd {
+    background-color: var(--color-error-solid);
+}
+
+.badge.notification {
+    top:0;
+    border: none;
+    width : 1.5rem;
```
Comment (vivekmehto) [inline Stylesheets/Components/badge.css]: the number inside the badge is not readable. you can increase the width and then use flex to center it or give some padding.

**Review by vivekmehto — COMMENTED** [review]
Suggested some changes apart from that It's awesome.

File: Stylesheets/Components/badge.css (hunk 2 above)
Comment (anushasunil) [reply Stylesheets/Components/badge.css]: oh okay. I'll rectify it 👍

File: Stylesheets/Components/badge.css (hunk 3 above)
Comment (anushasunil) [reply Stylesheets/Components/badge.css]: okay, working on it. Thankyou!

**Comment by anushasunil** [comment]
updated with the changes suggested

https://user-images.githubusercontent.com/90134948/152341876-2d93f597-c4de-4a73-a8c6-25d7e2845ec6.mp4

**Review by vivekmehto — APPROVED** [review]
Now looks perfect.

**Review by raman1999 — COMMENTED** [review]
Looks good,  you can merge now

**Pull request merged**